sdk/communication/AzureCommunication/Tests/CommunicationTokenCredentialTests.swift

```diff
@@ -45,7 +45,8 @@ class CommunicationTokenCredentialTests: XCTestCase {
         "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjEwMH0.1h_scYkNp-G98-O4cW6KvfJZwiz54uJMyeDACE4nypg"
 
     private var fetchTokenCallCount: Int = 0
-
+    private let timeout: TimeInterval = 18.0
```

Comment:
why 18? instead of a round number? why we need to bump up the time out to a such a high number , 18?

Reply:
I don't think it needs to be so high I was just trying large numbers to make sure the failures were not because of timeouts

Reply:
can we try with 10 instead of 18?